pkg/storegateway/gateway.go

```diff
@@ -62,6 +65,39 @@ type Config struct {
 
 	EnabledTenants  flagext.StringSliceCSV `yaml:"enabled_tenants"`
 	DisabledTenants flagext.StringSliceCSV `yaml:"disabled_tenants"`
+
+	// Hedged Request
+	HedgedRequest HedgedRequestConfig `yaml:"hedged_request"`
+}
+
+type HedgedRequestConfig struct {
```

Comment:
Can we put `HedgedRequestConfig`  and relevant methods to `pkg/storage/bucket`? So that it is easier to reuse the same config to support other components in the future if needed

Reply:
Ok, it sounds good.